storage/rocksdb/rdb_threads.cc

```diff
@@ -61,17 +64,6 @@ int Rdb_thread::create_thread(const std::string &thread_name
   int err = mysql_thread_create(background_psi_thread_key, &m_handle, nullptr,
                                 thread_func, this);
 
-  if (!err) {
```

Comment:
Please log a task to restore if you haven't done so already - it this feature is useful